my_init.d/prepare-aiidalab.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 set -em
 
-# For backwards compatibility
-ln -sf /home/${SYSTEM_USER} /project 
```

Comment:
just to understand: is this causing the issue (and if so, why?)

Reply:
Yes, this is causing the issue, because the link will be created a second time within the target directory (the link source) when this command is executed twice.

You can test this yourself, by executing the following commands in a terminal:

```bash
mkdir ~/aiidalab-docker-stack-issue-225
cd ~/aiidalab-docker-stack-issue-225
mkdir -p home/aiida
ln -sf home/aiida project  # fine
ln -sf home/aiida project # creates loop
```

```console
$ ls -l home/aiida
lrwxrwxrwx 1 sadorf sadorf 10 Jan  5 18:30 aiida -> home/aiida
```
It is not clear to me why this issue has only occurred "recently" since this command has been around for a long time, but I am confident that it is the cause of the problem.

Reply:
> It is not clear to me why this issue has only occurred "recently" since this command has been around for a long time, but I am confident that it is the cause of the problem.

Because those "loops" were revealed by the `find` methods added to the docker startup files quite recently.